Sources/Views/InputTextView.swift

```diff
@@ -118,31 +155,46 @@ open class InputTextView: UITextView {
         layer.cornerRadius = 5.0
         layer.borderWidth = 1.25
         layer.borderColor = UIColor.lightGray.cgColor
-
-        addSubviews()
-        addConstraints()
+        addObservers()
+        addPlaceholderLabel()
     }
-
-    private func addSubviews() {
-
+    
+    /// Adds the placeholderLabel to the view and sets up its initial constraints
+    private func addPlaceholderLabel() {
```

Comment:
Probably better named `setupPlaceholderLabel` now because it does more than just add?